src/linker/implementation_metadata.yaml

```diff
@@ -18,35 +18,29 @@ person_linkage_spark_cluster:
 # development dummies
 step_1_python_pandas:
   step: step_1
-  image_path: /mnt/team/simulation_science/priv/engineering/er_ecosystem/images/
-  script_cmd: python src/linker/steps/dev/python_pandas/dummy_step.py
-  name: python_pandas
+  image_path: /mnt/team/simulation_science/priv/engineering/er_ecosystem/images/python_pandas.sif
+  script_cmd: python dummy_step.py
```

Comment:
```suggestion
  script_cmd: python /dummy_step.py
```

I think making these paths absolute will decrease the chance we make a mistake and don't notice. Same for R below.